tests/ClientTest.php

```diff
@@ -107,6 +107,42 @@ public function testCaptureException(): void
         $client->captureException($exception);
     }
 
+    /**
+     * @dataProvider captureExceptionWithExcludedExceptionsDataProvider
+     */
+    public function testCaptureExceptionWithExcludedExceptions(bool $shouldCapture, string $excluded, \Throwable $thrown): void
```

Comment:
Please rename this method to `testCaptureExceptionDoesNothingIfExcludedExceptionsOptionMatches` (so that it's a bit more descriptive). Also rename the parameter `$excluded` to `$excludedExceptions` and `$thrown` to `$exception`